test/specs/commonTests/implementsShorthandProp.tsx

```diff
@@ -18,34 +18,47 @@ export default Component => {
     const { mapsValueToProp } = options
     const { displayName } = ShorthandComponent
 
+    /**
+     * @param {ReactWrapper} wrapper [mounted Component]
+     * @param {(n: ReactWrapper) => boolean} propsCheckFn [boolean function that applies a boolean check on a mounted ShorthandComponent]
+     * return true if the ShorthandComponent that matches propsCheckFn is found
+     */
+    const testShorthandComponentProps = (
+      wrapper: ReactWrapper,
+      propsCheckFn: (n: ReactWrapper) => boolean,
```

Comment:
it is hard to reason about what does `n` stands for with only its name being given :)

Reply:
N/A anymore